cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -195,6 +174,41 @@ body {
             font-size: unit( @base-font-size-px / @base-font-size-px, em );
             font-weight: normal;
         }
+
+        // Hide the ">" icon next to menu items on desktop…
+        &-link {
+            .cf-icon-svg {
+                display: none;
+            }
+        }
+
+        // …however, show the icons in featured links.
+        &_content-list__featured {
+            .cf-icon-svg {
+                display: inline-block;
+            }
+        }
+
+        // Featured menu content is link items with icons.
+        &-list__featured {
+
+            & .cf-icon-svg {
+                display: inline;
+                color: @pacific;
+                margin-right: 10px;
+            }
+        }
+
+        &-list__featured li {
+            // Align the icon and text.
+            display: flex;
```

Comment:
I think you're trying to get the external link icon on the same line? You can delete this property and set `width: 100%` on the anchor link. See below.